### PR TITLE
Add stats subcommand

### PR DESCRIPTION
**Pull request by apotterri**
Add the `stats` subcommand to show some simple statistics about a set of
app maps.

### CODE REVIEW

File: cmd/upload.go [hunk 1]
```diff
@@ -169,10 +132,12 @@ func NewUploadCommand(options *UploadOptions, metadataProviders []metadata.Provi
 				fileTiming.Start("uploading")
 				resp, err := api.CreateScenario(application, bytes.NewReader(data))
 				if err != nil {
-					return fmt.Errorf("failed uploading %s: %w", scenarioFile, err)
-				}
+					fmt.Fprintf(os.Stderr, "warning, failed uploading %s: %s\n", scenarioFile, err)
+				} else {
+					// fmt.Fprintf(os.Stderr, "[%w] created scenario %s\n", time.Now().String(), resp.UUID)
```
Comment (kgilpin) [inline cmd/upload.go:137]: Is this meant to be commented out?

**Review by kgilpin — COMMENTED** [review]
Can you add an example of usage to the PR description? Also, a documentation snippet and an example in the README would be a big plus.

**Comment by apotterri** [comment]
@kgilpin The usage examples were pretty big, and seemed like they belonged in the README. LMK if you'd like more in the PR.

**Review by kgilpin — APPROVED** [review]
Nice!

**Pull request merged**